src/components/Feed/WrapToken.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  GU,
+  Help,
+  useLayout,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
+
+import { useAppState } from '../../providers/AppState'
+
+import { formatTokenAmount } from '../../utils/token-utils'
+
+import wrappedIcon from '../../assets/wrappedIcon.svg'
+import unwrappedIcon from '../../assets/unwrappedIcon.svg'
+
+function WrapToken({ onUnwrapToken, onWrapToken }) {
+  const { layoutName } = useLayout()
+  const {
+    accountBalance: gardenTokenBalance,
+    stakeToken,
+    wrappableAccountBalance,
+    wrappableToken,
+  } = useAppState()
+
+  const theme = useTheme()
+  const compactMode = layoutName === 'small' || layoutName === 'medium'
+  const tabletMode = layoutName === 'medium'
+
+  return (
+    <Box
```

Comment:
```suggestion
    <Box
      padding={2.5 * GU}
```

Reply:
i don't think this one is needed since we are going to set a padding of 20 while we have in all our other boxes 24